App/screens/ProofRequest.tsx

```diff
@@ -139,11 +174,12 @@ const CredentialOffer: React.FC<Props> = ({ navigation, route }) => {
             text1: t('ProofRequest.RejectingProof'),
           })
           try {
-            // await agent.proofs.rejectPresentation(id)
-            navigation.goBack()
-          } catch {
+            await agent?.proofs?.declineRequest(proof?.id)
+          } catch (e: unknown) {
+            // console.error(e)
```

Comment:
```suggestion
```

Reply:
Will remove these. Something for us to think about is whether we need a logging type of module in the wallet?